### ISSUE TITLE
System crash when pressing new backup for system saves

### ISSUE THREAD

**Issue by WallK** [J-D-K/JKSV#33]
In version 05.20.2020 when trying to backup ACNH, for example, system crashes after pressing A on New option

**Comment by J-D-K** [comment]
That sounds like the keyboard doesn't have enough RAM to open. Are you using a lot of sys modules?

**Comment by WallK** [comment]
No, just the basic stuff I think
Older version (04.10.2020) works alright

**Comment by J-D-K** [comment]
Does holding L or R while selecting new work?

**Comment by WallK** [comment]
Yes
Was it auto naming by default in previous version?

**Comment by J-D-K** [comment]
If it was run through the album it did auto name in the previous version. It's definitely the keyboard failing to launch. Most people say a clean atmosphere reinstall fixes it. The last time I helped someone with this, they had way too many sys modules and that's why it failed. I only have Freebird, which I made sure uses as little as memory as possible. I can check if there is a solution, once the keyboard function is called, I think it's out of my hands.
